script/backport-patchset.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python
+
+import os
+import re
+import sys
+import urllib2
+
+SOURCE_ROOT = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
+PATCHES_DIR = os.path.join(SOURCE_ROOT, 'patches')
+
+def main():
+  if len(sys.argv) < 3:
+    print("Usage: backport-patchset.py [code-review-id] [patch-name]")
+    return 1
+  codereview_url = "https://codereview.chromium.org/" + sys.argv[1]
+
+  codereview_response = urllib2.urlopen(codereview_url)
```

Comment:
`main()` is really huge now, maybe we could extract getting a diff by a `code-review-id` to a function?